Cold-Gas-Thruster/Cold Gas Thruster PDD.tex

```diff
@@ -0,0 +1,469 @@
+\nomenclature{PDD}{Project Design Document}
+\nomenclature{CGT}{Cold Gas Thruster}
+\nomenclature{RCS}{Reaction Control System}
+\nomenclature{$I_{sp}$}{Specific Impulse [s]}
+\nomenclature{$I_t$}{Total Impulse [Ns]}
+\nomenclature{$g_0$}{Standard gravity [m/s^2]}
+\nomenclature{$m_p$}{Propellant Mass [kg]}
+\nomenclature{$t_f$}{Final time [s]}
+\nomenclature{$t_i$}{Initial time [s]}
+%\nomenclature{\gamma}{Specific Heat Ratio}
+% Below are examples of using nomenclature for math symbols and constants or units
+% \nomenclature{$\dot{m}$}{Mass flow rate
+%   \nomunit{\,\si{\kilo\gram\per\second}}}
+% \nomenclature{$c$}{Speed of light
+%  \nomunit{\,\SI{2.9979e8}{\meter\per\second}}}
+\printnomenclature{}
+
+
+% HELPFUL HINT: If you get the warning ``Command terminated with space.'' when using a \command try placing ``%'' or ``{}'' immediately following the command.
+
+% The sections included here are required. Additional sections and subsections may be added as necessary.
+\section{Introduction}
+\label{sec:introduction}
+  % The introduction is a place to give background and context before diving into the subject matter.
+  % Establish context forBBB the work you are about to propose and the main ideas of the proposition itself.
+
+\IEEEPARstart{T}{he} objective of this project is create a cold gas thruster. The CGT will allow the team to gain experience in designing, building, and testing propulsion systems.
+The type of nozzle that will be tested is a minimum length supersonic nozzle. Since there is ambient pressure, the nozzle will be optmized for ambient conditions.
+
+\section{Primary Objective}
+\label{sec:primary-obj}
+  % At the end of the day, whether the project ``succeeds'' or ``fails'' is judged against the objectives it sought to meet.
+  % Note that results that contradict expectations/hypotheses are not failures if the scientific \& engineering methods are followed along the way.
+  % Sometimes our expectations are wrong and that can be just as successful as getting data we thought we'd see.
+  % What matters are what questions you intend to answer.
+  % This is the main purpose or main goal the project hopes to achieve.
+The primary objective of this project is to create a cold gas thruster for on-ground experimentation. The goal for the data is to be able to design a nozzle for a small propulsion system, where boundary
+layer effects are dominent, but also to achieve an specific impulse of 50s. Using the thrust over time curves, the specific impulse can be
+ calculated numerically using MATLAB\@. The data will be transferred and analyzed there using the following equations as a basis.
+
+
+\begin{equation} \label{specific impulse}
+  I_t    = \int_{t_i}^{t_f} F_{thrust} dt
+\end{equation}
+
+\begin{figure}[h]
+  \includegraphics[width=\linewidth]{figs/cold-gas-thruster.png}
+  \caption{This is the Solidworks Assembly of the current design of the SPEX cold gas thruster. The piping interface to the nozzle is omitted for clarity.}
```

Comment:
* Remove `This is`
* Change `current design` to something more objective and "timeless" I guess. Or just say it was designed in Spring 2017 or something
* What is "the piping interface"?